### ISSUE TITLE
Sensors data is not exported using InfluxDB2 exporter

### ISSUE THREAD

**Issue by philippfriese** [nicolargo/glances#2856]
**Describe the bug**
Exporting data from the `sensors` plugin using the `influxdb2` exporter fails in the `_normalize` call:

```
$ glances --export influxdb2 --quiet --debug
[...]
2024-06-29 11:53:08,978 -- DEBUG -- Cannot export sensors stats to InfluxDB ('SensorType.CPU_TEMP')
```

**Additional Information**
This line is triggered in [`exports/glances_influxdb2/__init__.py:L169`](https://github.com/nicolargo/glances/blob/ccfd8f0aa95238d86f1418fd8ffff32c7014462b/glances/exports/glances_influxdb2/__init__.py#L169). The error is thrown in [`self._normalize`](https://github.com/nicolargo/glances/blob/ccfd8f0aa95238d86f1418fd8ffff32c7014462b/glances/exports/glances_influxdb2/__init__.py#L166), more specifically in [`__init__.py:L151`](https://github.com/nicolargo/glances/blob/ccfd8f0aa95238d86f1418fd8ffff32c7014462b/glances/exports/glances_influxdb2/__init__.py#L151), which tries to run `fields.pop(fields[k])` (`k == 'type'`). `fields` contains the entry `{'type': <SensorType.CPU_TEMP: 'temperature_core'>, ...}` and notably does **not** contain a tuple with key `temperature_core`.
This can be seen when calling `self._normalize` inside the exception handler and thus triggering a second exception, crashing the whole stack: 

<details><summary>Modification</summary>

```diff
@@ -166,7 +166,8 @@ class Export(GlancesExport):
             try:
                 self.client.write(self.bucket, self.org, self._normalize(name, columns, points), time_precision="s")
             except Exception as e:
                 # Log level set to debug instead of error (see: issue #1561)
                 logger.debug(f"Cannot export {name} stats to InfluxDB ({e})")
+                #logger.error(self._normalize(name, columns, points))
             else:
                 logger.debug(f"Export {name} stats to InfluxDB")
```

</details>
<details><summary>Traceback</summary>

```
$ glances --export influxdb2 --quiet --debug
Exception in thread Thread-4 (update):
Traceback (most recent call last):
  File "/home/arch/venvs/venvglances/lib/python3.12/site-packages/glances/exports/glances_influxdb2/__init__.py", line 168, in export
    self.client.write(self.bucket, self.org, self._normalize(name, columns, points), time_precision="s")
                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/arch/venvs/venvglances/lib/python3.12/site-packages/glances/exports/glances_influxdb2/__init__.py", line 153, in _normalize
    fields.pop(fields[k])
KeyError: 'SensorType.CPU_TEMP'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.12/threading.py", line 1073, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.12/threading.py", line 1010, in run
    self._target(*self._args, **self._kwargs)
  File "/home/arch/venvs/venvglances/lib/python3.12/site-packages/glances/exports/export.py", line 179, in update
    self.export(plugin, export_names, export_values)
  File "/home/arch/venvs/venvglances/lib/python3.12/site-packages/glances/exports/glances_influxdb2/__init__.py", line 172, in export
    logger.error(self._normalize(name, columns, points))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/arch/venvs/venvglances/lib/python3.12/site-packages/glances/exports/glances_influxdb2/__init__.py", line 153, in _normalize
    fields.pop(fields[k])
KeyError: 'SensorType.CPU_TEMP'
```

</details>

**To Reproduce**
1. Start Glances with the following options: `glances --export influxdb2 --quiet --debug`
2. Get message `DEBUG -- Cannot export sensors stats to InfluxDB ('SensorType.CPU_TEMP')`

**Expected behavior**
Data from the sensor data to appear in the database.

**Environment**
 - Operating System: Arch Linux Kernel 5.10.2, aarch64
 - Python version 3.12.3
 - Glances & psutil versions: `Glances version: 4.1.0, Glances API version: 4, PsUtil version: 6.0.0`
 - Installed via pip (`pip install glances`)
 - Glances test: [glances_test.txt](https://github.com/user-attachments/files/16039544/glances_test.txt)

**Configuration**
The configuration file has not been modified in any way, except the InfluxDB2 credentials. Specifically, the sensor configuration section completely matches the provided template.

**Comment by nicolargo** [comment]
May be related to https://github.com/nicolargo/glances/issues/2788

@RazCrimson Can you have a look ?

**Comment by nicolargo** [comment]
@RazCrimson Forget, it is not related...

I am on it.

**Comment by nicolargo** [comment]
@philippfriese just pushed a patch on the develop branch. Can you give it a try ?

**Comment by philippfriese** [comment]
@nicolargo Just checked, works now on my end as well. Thank you!

**Comment by nicolargo** [comment]
Glances 4.1.1 (with the patch) is available in Pypi.

Thanks again for the issue report / test @philippfriese

**Comment by vajonam** [comment]
@nicolargo  

Curious if this is fixed in 4.1,2 ? I am using ubuntu-latest-full. I dont see this in the influxdb2, missing the sensors fields, I see number of other things including containers.
